packages/dcos-integration-test/extra/test_endpoints.py

```diff
@@ -191,6 +209,28 @@ def test_if_overlay_master_agent_is_up(cluster):
     except KeyError as ex:
         raise AssertionError("Could not find key 'subnet':" + str(ex)) from ex
 
+    # Pop mesos and docker bridge if they have been configured on the
```

Comment:
One high level question.  I was trying to understand the exact assertion as an integration test here.

If we all we care about is *agent_overlay_agent_response* should contain either `mesos_bridge` or `docker_bridge`  is it better to assert that straight instead of this `KeyError and assert pattern` ?